src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js

```diff
@@ -1282,7 +1282,7 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
 
                     <DropdownItem
                       onClick={() => {
-                        if (resetBinsDisabled) return;
+                        if (!isCustomized) return;
```

Comment:
could this not be a positive that wraps everything else?

Reply:
did this on the card refactor

```if (binsAreCustom) {
          dispatchUpdateClinicalVariable({
....etc
```

Reply:
Meaning you're already taking care of this in a later commit? @samrichca